source/people/aaswenson.rst

```diff
@@ -0,0 +1,29 @@
+Alexander Swenson
+=============
+
+Contact Information
+-------------------
+| `E-mail <aaswenson@wisc.edu>`_
+
+----
```

Comment:
I am not sure  there is a rule about the `---` line, some people have it, some don't... It just create a separation between contact and main text